compiler/rustc_middle/src/mir/interpret/mod.rs

```diff
@@ -125,10 +125,12 @@ use std::io::{Read, Write};
 use std::num::NonZero;
 use std::sync::atomic::{AtomicU32, Ordering};
 
+use smallvec::{smallvec, SmallVec};
+use tracing::debug;
```

Comment:
Why is this import needed now? The diff doesn't otherwise mention `debug!`...

Reply:
It was a rebasing mix-up relating to the third commit removing `extern crate tracing`. I have removed it.

Reply:
Because `debug!` is used, *and* there is a `#[macro_use] extern crate tracing`, so there are no warnings/errors whether the `use` is present or not.